src/core/Akka.Streams.Tests/Dsl/AsyncEnumerableSpec.cs

```diff
@@ -26,19 +26,20 @@
 
 namespace Akka.Streams.Tests.Dsl
 {
-#if NETCOREAPP
```

Comment:
You need to add this back @eaba  - your new `#if` is what's causing the compilation error.